### DISCUSSION TITLE
Text hover not working

### DISCUSSION THREAD

**Discussion by wonrax** [nvim-java/nvim-java#203] [Q&A]
I installed nvim-java following the instructions, however text hovering does not seem to work:

```
method textDocument/hover is not supported by any of the servers registered for the current buffer
```

Other LSP features like go to definition or completion work perfectly.

**Comment by diegoadborges** [comment]
I had the same problem, this solved it for me

[#212 ](https://github.com/nvim-java/nvim-java/issues/212#issuecomment-2198535905)
